fv3core/pace/fv3core/stencils/delnflux.py

```diff
@@ -1,29 +1,41 @@
 from typing import Optional
 
 import gt4py.gtscript as gtscript
-import numpy as np
 from gt4py.gtscript import (
     __INLINED,
-    FORWARD,
     PARALLEL,
     computation,
     horizontal,
     interval,
     region,
 )
 
-import pace.dsl.gt4py_utils as utils
+import pace.util
 from pace.dsl.dace.orchestration import orchestrate
 from pace.dsl.stencil import StencilFactory, get_stencils_with_varied_bounds
 from pace.dsl.typing import FloatField, FloatFieldIJ, FloatFieldK
-from pace.util import X_DIM, Y_DIM, Z_DIM
-from pace.util.constants import X_INTERFACE_DIM, Y_INTERFACE_DIM
+from pace.util import X_DIM, X_INTERFACE_DIM, Y_DIM, Y_INTERFACE_DIM, Z_DIM
 from pace.util.grid import DampingCoefficients
 
 
-def calc_damp(damp4: FloatField, nord: FloatFieldK, damp_c: FloatFieldK, da_min: float):
-    with computation(FORWARD), interval(...):
-        damp4 = (damp_c * da_min) ** (nord + 1)
+def calc_damp(
+    damp_c: pace.util.Quantity, da_min: float, nord: pace.util.Quantity
+) -> pace.util.Quantity:
+    if damp_c.dims != nord.dims or damp_c.data.shape != nord.data.shape:
+        raise NotImplementedError(
+            "current implementation requires damp_c and nord to have "
+            "identical data shape and dims"
+        )
+    data = (damp_c.data * da_min) ** (nord.data + 1)
```

Comment:
Why is this moved away from gt4py stencil?

Reply:
Are all the removed stencils because it's called once in init only?

Reply:
Yes. I removed the gt4py stencils done for computing 1D values to avoid adding more complexity to the already complex make-a-3d-storage-to-compute-to-subset-into-a-1d-storage logic.